Networking/Networking/Model/SitePlugin.swift

```diff
@@ -0,0 +1,156 @@
+            throw SitePluginError.missingSiteID
+        }
+
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let plugin = try container.decode(String.self, forKey: .plugin)
+        let status = try container.decodeIfPresent(String.self, forKey: .status) ?? ""
+        let name = try container.decodeIfPresent(String.self, forKey: .name) ?? ""
+        let pluginUri = try container.decodeIfPresent(String.self, forKey: .pluginUri) ?? ""
+        let author = try container.decodeIfPresent(String.self, forKey: .author) ?? ""
+        let authorUri = try container.decodeIfPresent(String.self, forKey: .authorUri) ?? ""
+        let version = try container.decodeIfPresent(String.self, forKey: .version) ?? ""
+        let networkOnly = try container.decodeIfPresent(Bool.self, forKey: .networkOnly) ?? false
+        let requiresWPVersion = try container.decodeIfPresent(String.self, forKey: .requiresWPVersion) ?? ""
+        let requiresPHPVersion = try container.decodeIfPresent(String.self, forKey: .requiresPHPVersion) ?? ""
+        let textDomain = try container.decodeIfPresent(String.self, forKey: .textDomain) ?? ""
+
+        let descriptionContainer = try container.nestedContainer(keyedBy: DescriptionCodingKeys.self, forKey: .description)
+        let descriptionRaw = try descriptionContainer.decodeIfPresent(String.self, forKey: .raw) ?? ""
+        let descriptionRendered = try descriptionContainer.decodeIfPresent(String.self, forKey: .rendered) ?? ""
+
+        self.init(
+            siteID: siteID,
+            plugin: plugin,
+            status: status,
+            name: name,
+            pluginUri: pluginUri,
+            author: author,
+            authorUri: authorUri,
+            descriptionRaw: descriptionRaw,
+            descriptionRendered: descriptionRendered,
+            version: version,
+            networkOnly: networkOnly,
+            requiresWPVersion: requiresWPVersion,
+            requiresPHPVersion: requiresPHPVersion,
+            textDomain: textDomain
+        )
+    }
+}
+
+/// Defines all of the SitePlugin CodingKeys.
+///
+private extension SitePlugin {
+
+    enum CodingKeys: String, CodingKey {
+        case plugin              = "plugin"
+        case status              = "status"
+        case name                = "name"
+        case pluginUri           = "plugin_uri"
+        case author              = "author"
+        case authorUri           = "author_uri"
+        case description         = "description"
+        case version             = "version"
+        case networkOnly         = "network_only"
+        case requiresWPVersion   = "requires_wp"
+        case requiresPHPVersion  = "requires_php"
+        case textDomain          = "textdomain"
+    }
+
+    enum DescriptionCodingKeys: String, CodingKey {
+        case raw      = "raw"
+        case rendered = "rendered"
+    }
+}
+
+// MARK: - Comparable Conformance
+//
+extension SitePlugin: Comparable {
```

Comment:
Do we really need the `Comparable` conformance? I think that the `Equatable` conformance should be enough, but perhaps there are cases that I have not considered where `Comparable` is needed?

Reply:
Hmmmm. Probably not. I'll delete it.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/8e320fa88e27122e36d57a3de6e396bc6ec0ce37

Reply:
I think that you can use just the `Equatable` conformance without declaring a custom `func ==` 😉 because, in theory, you should use all the fields, like the `author` (maybe there are two plugins with the same name, but with a different author? I don't know if this is possible).

Reply:
TIL. Deleted it in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/2dcffd48d2bc4143f6fdfd095b8a7e20076072d1